great_expectations/rule_based_profiler/profiler.py

```diff
@@ -111,88 +131,33 @@ def __init__(
     def profile(
         self,
         *,
-        validator=None,
-        batch=None,
-        batches=None,
-        batch_request=None,
```

Comment:
This is above my pay grade, but I liked when there were multiple ways to specify batches. Did we decide to remove all of these?

Reply:
@anthonyburdi -- there was some discussion that having `batch_ids` was not needed.  But overall, I heard one comment that it would be great to make the interface as close to that of `great_expectations/profile/user_configurable_profiler.py` as possible.  I found that it would be easier to just put the `Validator` into the constructor (and `DataContext`) and then derive all possibilities from there.  We are not using the multiple ways now, and with the removal of `batch_ids` from `DomainBuilder` and `ParameterBuilder` I felt that it would be cleaner to remove those arguments.  If in the future we encounter use cases for them, we can easily add them back.  Happy to discuss.  Thanks.